portfolio/src/main/webapp/script.js

```diff
@@ -56,15 +56,23 @@ function interests() {
 
 /** Creates a map and adds it to the page. */
 function loadMap() {
-  const stanfordGatesBldgLatLng = {lat:37.430175, lng:-122.173388};
-  const map = new google.maps.Map(
-      document.getElementById('map'),
-      {center: stanfordGatesBldgLatLng, zoom: 16});
-  const marker = new google.maps.Marker({
-        position: stanfordGatesBldgLatLng, 
+    var desiredLocationLatLng = {lat: 46.233950, lng: 6.055801};
+    if (document.getElementById("locations").value === "Ramen") {
```

Comment:
So since the map is being loaded at body load time, the `select` element would always be empty? Am I right? No action needed here, but i think you want to pull the maps centering work into a separate method so that you can re-focus the map every time the `location` changes.